dashboard/src/shared/utils.ts

```diff
@@ -96,3 +98,57 @@ export function getPluginIcon(plugin?: Plugin | string) {
     }
   }
 }
+
+export function getPluginName(plugin?: Plugin | string) {
+  // Temporary case while operators are not supported as kubeapps apis plugin
+  if (typeof plugin === "string") {
+    switch (plugin) {
+      case "chart":
+        return "Helm";
+      case "helm":
+        return "Helm";
+      case "operator":
+        return "Operator";
+      default:
+        return "unknown plugin";
+    }
+  } else {
+    switch (plugin?.name) {
+      case PluginNames.PACKAGES_HELM:
+        return "Helm";
+      case PluginNames.PACKAGES_FLUX:
+        return "Flux";
+      case PluginNames.PACKAGES_KAPP:
+        return "Carvel";
```

Comment:
> (4) (first letter capitalized),

Done :)

Reply:
> > I don't see any numbers in the ishihara 9 test
> 
> I can replicate it with protanopia (no red) and deuteranopia (no green), so next time I'll double-check all the UI decisions with (at least) these two options.

Heh - Thanks Antonio, but I've never noticed any disadvantage caused by it.